log/logger.go

```diff
@@ -0,0 +1,27 @@
+package log
+
+import (
+	"fmt"
+
+	kitlog "github.com/go-kit/kit/log"
+)
+
+// Logger is what any Tendermint library should take.
+type Logger interface {
+	Debug(msg string, keyvals ...interface{}) error
```

Comment:
No, we are not. Still it's good to have in case you want to check why logger does not produce output or write tests. 

Originally from https://github.com/go-kit/kit/blob/master/log/log.go#L11
Error could be like https://github.com/go-kit/kit/blob/master/log/json_logger.go#L34 json encoding error